frontend/src/metabase/palette/hooks/useCommandPalette.tsx

```diff
@@ -256,6 +261,10 @@ export const useCommandPalette = ({
   ]);
 
   const adminActions = useMemo<PaletteAction[]>(() => {
+    if (!isAdmin) {
```

Comment:
I think this is too restrictive, because it means that people with access to tools / troubleshooting, or access to manage DBs, are now missing links. I'll submit a PR that fixes this, it's a small adjustment